napari/components/camera.py

```diff
@@ -13,7 +15,8 @@ class Camera(EventedModel):
     Attributes
     ----------
     center : 3-tuple
-        Center of the camera. In 2D viewing the last two values are used.
+        Center of rotation for the camera.
```

Comment:
nitpick unrelated to this PR - can separate out if anyone cares!

Reply:
fine with me